notebook/starter_notebook.ipynb

```diff
@@ -0,0 +1,981 @@
+   "metadata": {},
+   "source": [
+    "Here, the fit() function generate the trained model with the given dataset.\\\n",
+    "Score function shows how \"well\" the model is predicting. \\\n",
+    "0 shows the model is completely random(or not finding any correlation between independent and dependent variables)\\\n",
+    "while 1 is saying the model found the complete relationship between independent and dependent variables."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 56,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "0.5850552605968369"
+      ]
+     },
+     "execution_count": 56,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "model = LinearRegression().fit(X_train, y_train)\n",
+    "model.score(X_train, y_train)\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 57,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "0.5843728155524242"
+      ]
+     },
+     "execution_count": 57,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "model.score(X_test, y_test)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Model has slightly lower score on test set compared to the result of training set. Why?"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This how to view coefficients(which shows that 'relationship', or how much each independet variable affects the depedent variable)\\\n",
+    "and interncept(y-intercept in this case because we are trying to find the 'line of best fit')."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 46,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "[ 0.6968879   1.92073051 10.49583148  0.24035778 -0.67270173 -0.30180624\n",
+      "  0.55927075]\n",
+      "2.4008549738556626\n"
+     ]
+    }
+   ],
+   "source": [
+    "print(model.coef_)\n",
+    "print(model.intercept_)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now this is the time to see the actual prediction values that our model is generating."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 60,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "array([ 8.2246501 , 11.13004033, 10.81047255, ...,  9.80954453,\n",
+       "       15.67641267, 13.82234011])"
+      ]
+     },
+     "execution_count": 60,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "prediction = model.predict(X_train)\n",
+    "prediction\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "One of the most common way to numerize the 'error'(or how off the model is) root mean squared error(RMSE).\\\n",
```

Comment:
replace with "One of the most common ways to quantify the error of the model is the root mean squared error (RMSE)."